timetracker/entries/templates/client_list.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+    <table>
+        <tr>
+            <th>Client Names</th>
+        </tr>
+
+        {% if client_list %}
+
+            {% for client in client_list %}
+                <tr>
+                    <td><a href="{% url 'client_detail' client.id %}">{{ client.name }}</a></td>
+                    <td><a href="{% url 'client_summary' client.id %}">Summary</a></td>
+                </tr>
+            {% endfor %}
+        {% else %}
```

Comment:
Pro tip: the {% for %} block tag allows for a {% empty %} tag inside so you can merge this if/else inside the for.